saleor/plugins/avatax/tests/test_avatax.py

```diff
@@ -485,6 +487,217 @@ def test_taxes_need_new_fetch_uses_cached_data(
     assert not taxes_need_new_fetch(checkout_data, str(checkout_with_item.token))
 
 
+AVALARA_TAX_DATA = {
+    "id": 0,
+    "companyId": 123,
+    "date": "2020-12-28",
+    "paymentDate": "2020-12-28",
+    "status": "Temporary",
+    "type": "SalesOrder",
+    "summary": [
+        {
+            "country": "US",
+            "region": "NE",
+            "jurisType": "State",
+            "jurisCode": "31",
+            "jurisName": "NEBRASKA",
+            "taxAuthorityType": 45,
+            "stateAssignedNo": "",
+            "taxType": "Sales",
+            "taxSubType": "S",
+            "taxName": "NE STATE TAX",
+            "rateType": "General",
+            "taxable": 36.94,
+            "rate": 0.055,
+            "tax": 2.03,
+            "taxCalculated": 2.03,
+            "nonTaxable": 0.0,
+            "exemption": 0.0,
+        }
+    ],
+}
+
+
+@pytest.mark.vcr
+def test_get_checkout_tax_rate(
```

Comment:
All added tests are marked with `@pytest.mark.vcr`. Those tests use mocks instead of an external connection. Should we remove this mark for all tests than not use an external connection?